corehq/apps/data_interfaces/tasks.py

```diff
@@ -244,3 +250,47 @@ def _get_repeat_record_ids(
             return [r['id'] for r in queryset.values('id')]
         else:
             return list(iter_repeat_record_ids_by_repeater(domain, repeater_id))
+
+
+@task
+def bulk_case_reassign_async(domain, user_id, owner_id, download_id, report_url):
+    task = bulk_case_reassign_async
+    case_ids = DownloadBase.get(download_id).get_content()
+    DownloadBase.set_progress(task, 0, len(case_ids))
+    user = CouchUser.get_by_user_id(user_id)
+    submission_handler = SubmitCaseBlockHandler(
+        domain, None, None, user, None, throttle=True
+    )
+    for idx, case_id in enumerate(case_ids):
+        submission_handler.add_caseblock(
+            RowAndCase(idx, CaseBlock(case_id, owner_id=owner_id))
+        )
+        DownloadBase.set_progress(task, idx, len(case_ids))
+    submission_handler.commit_caseblocks()
+    DownloadBase.set_progress(task, len(case_ids), len(case_ids))
+    result = submission_handler.results.to_json()
+    result['success'] = True
+    result['case_count'] = len(case_ids)
+    result['report_url'] = report_url
+
+    def _send_email():
+        context = {
+            'case_count': len(case_ids),
+            'report_url': report_url,
+        }
+        text_content = """
+        {case_count} cases were reassigned. The list of cases that
+        were reassigned are <a href='{report_url}'>here</a>.
+        It's possible that in the report the owner_id is not yet
+        updated, you can open individual cases and confirm
+        the right case owner, the report gets updated with a slight delay.
+        """.format(**context)
+        send_HTML_email(
+            "Reassign Cases Complete on {domain}- CommCare HQ",
+            user.get_email(),
+            render_to_string("data_interfaces/partials/case_reassign_complete_email.html", context),
```

Comment:
I think `data_interfaces/partials/case_reassign_complete_email.html` should be `data_interfaces/partials/case_reassign_status.html`. I don't see a `case_reassign_complete_email.html` file

Reply:
Thanks for pointing out, I forgot to commit the file